### PR TITLE
fix: added preview link

### LINKED ISSUE #494: [BUG]: Link preview is not available.
### Description:

The link preview is not visible whenever a user sends a link; the preview should be visible.

### Steps to reproduce:

1. Set up the EmbeddedChat and Open Storybook.
2. Send a message along with a link.
3. Notice that only the link is available with no preview. However, in the Rocket Chat app, we do have the link preview option.

### Expected behavior:

The link preview must be visible and allow the user to click on it to open the link in a separate window.

### Actual behavior:

The link preview is not available.

https://github.com/RocketChat/EmbeddedChat/assets/78961432/4d988041-656e-47b9-963b-21789f894f9b

### PR DESCRIPTION
**Pull request by Spiral-Memory**
# Brief Title
Added a feature to display link previews, which were not visible earlier.

## Acceptance Criteria fulfillment

- [x] Added link preview feature.
- [x] Included an option to show/hide the link preview.
- [x] Introduced the necessary icon.

Fixes #494 

## Video/Screenshots

https://github.com/RocketChat/EmbeddedChat/assets/78961432/ee0eb462-36d1-423c-ac7c-3f1cd0d51d20

Added another video to demonstrate that users can indeed click on the image, link, or the source to navigate to the link, just like in Rocket Chat.

https://github.com/RocketChat/EmbeddedChat/assets/78961432/3da6a561-42d7-4822-a738-058a2f8e81b9

### CODE REVIEW

**Comment by Spiral-Memory** [comment]
Ready to review ..

**Comment by Spiral-Memory** [comment]
Ready for review ..

**Comment by Spiral-Memory** [comment]
Ready for review ..

File: packages/react/src/components/Message/Message.js [hunk 1]
```diff
@@ -157,6 +158,7 @@ const Message = ({
 
   const isStarred = message.starred?.find((u) => u._id === authenticatedUserId);
   const shouldShowHeader = !sequential || (!showAvatar && isStarred);
+  console.log(message);
```
Comment (sidmohanty11) [inline packages/react/src/components/Message/Message.js:161]: please remove this

**Review by sidmohanty11 — APPROVED** [review]
LGTM! Thanks @Spiral-Memory !! Just added a minor comment

**Pull request merged**